thrust/testing/cuda/scan.cu

```diff
@@ -186,6 +203,20 @@ void TestScanCudaStreams()
   ASSERT_EQUAL(input, input_copy);
   ASSERT_EQUAL(output, result);
 
+  // inclusive scan with init and op
+  iter =
+    thrust::inclusive_scan(thrust::cuda::par.on(s), input.begin(), input.end(), output.begin(), 3, thrust::plus<T>());
+  cudaStreamSynchronize(s);
+
+  result[0] = 4;
```

Comment:
No change requested: I really do not like the style of setting individual elements of vectors. I have filed an issue for a contributor to clean all that up

Reply:
I agree!

Reply:
#2356